.github/workflows/release.yml

```diff
@@ -0,0 +1,96 @@
+name: release
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+env:
+  JOB_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+  SLACK_CHANNEL: "#apm-agent-dotnet"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    env:
+      DOCKER_IMAGE_NAME: "docker.elastic.co/observability/elastic-otel-collector-azure"
```

Comment:
**question**: what's the reason that the docker image does not match the name of the GitHub repository?

Reply:
This started as a demo repository to demonstrate how users can deploy their own configurations to Azure Container Repository for .NET. 

The scope has changed to include this pre-baked image for the Azure Container Sidecar UX. By publishing this we become part of a dropdown making it easier for users to start using our elastic collector on azure. 

Long term I think this might have to be moved back into the beats repos pending discussions with the ingest team.

Reply:
Thanks Martijn for the explanation 🙏